addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -1234,20 +1238,54 @@
 		
 		"307"	//Ullapool Caber
 		{
-			"desp"			"Ullapool Caber: {positive}100% explosion damage bonus, {negative}marked-for-death while active"
-			"attrib"		"414 ; 1.0"
+			"desp"			"Ullapool Caber: {neutral}Explosion deals massive damage as a true last resort killing yourself in the process, {negative}marked-for-death while active"
+			"attrib"		"414 ; 1.0 ; 207 ; 11.0"
+			
+			"spawn"
+			{
+				"Tags_SetEntProp"	//Make the caber fucking massive
+				{
+					"target"		"melee"
+					"type"			"float"
+					"prop"			"m_flModelScale"
+					"value"			"3.0"
+				}
+			}
 			
 			"attackdamage"
 			{
 				"filter"
 				{
-					"victimuber"	"0"
-					"damagetype"	"blast"
+					"damagetype"	"blast"	//Only when the caber detonates
 				}
 				
 				"params"
 				{
-					"multiply"		"2.0"	//Multiply explosion damage by 2
+					"multiply"		"3.0"	//Multiply explosion damage by 3
+				}
+				
+				"Tags_Explode"
```

Comment:
Should add target param to victim

Reply:
did this in a different way, made the filter consider melee and blast damage instead